docs/schema/merging.md

```diff
@@ -43,30 +43,30 @@ To convert a field's value in a release to a **versioned value**, you must:
 
 A **versioned value** thus describes a field's value in a specific release.
 
-For example, in the above worked example, the estimated value of the procurement was $1,000 in a release (`tender/value/amount` was `1000`). Following the steps above, the versioned value is:
+For example, in worked example 1 in [Updates and deletions](../guidance/build/merging), the estimated value of the procurement was 13,000 GHS (in the first release `tender/value/amount` was `13000`). Following the steps above, the versioned value is:
 
 ```json
 {
-  "releaseID": "ocds-213czf-000-00002-01-tender",
-  "releaseDate": "2016-01-01T09:30:00Z",
+  "releaseID": "ocds-uhveoc-144560-1601380494627-tender",
+  "releaseDate": "2020-10-21T13:02:26Z",
   "releaseTag": [
     "tender"
   ],
-  "value": 1000
+  "value": 13000
 }
 ```
```

Comment:
Rather than linking to the worked example and including only the versioned value on this page, I think the purpose of this example (illustrating the relationship between a release and a versioned release) would be better served by adding a `jsoninclude` directive showing the relevant fields in the release (`id`, `date`, `tag` and `tender.value.amount`).

Reply:
Noting that this comment is still to be addressed.

Reply:
I think that the purpose of the example is to illustrate the relationship between a release and a versioned release. Presently, the only JSON example on the page is the versioned release so if users want to see what the associated release looks like they need to follow the link to the updates and deletions worked example. My suggestion is to replace the link to the updates and deletions worked example with a `jsoninclude` directive showing the relevant fields in the release so that readers can see both the release and versioned release on the same page. Does that make sense?

Reply:
ooh, right I see, sorry it makes sense now. There's never been a release for this example, only a versioned.json file so I'll need to create an initial release for this.

Reply:
Just noting that I'll push this change and the other similar one once there's agreement about changing this to be fictional (see comment in https://github.com/open-contracting/standard/pull/1661#discussion_r1421936525

Reply:
should be fixed now

Reply:
Resolving as I've updated this example.